webapp/src/ts/modules/tasks/tasks.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="filters">
+  <!--div class="inner">
+    <mm-navigation></mm-navigation>
+  </div-->
+</div>
+
+<div class="inner">
+  <div class="inbox page">
+
+    <div id="tasks-list" class="col-sm-4 inbox-items left-pane">
+      <div class="selection-count" *ngIf="selectMode">
+        <span *ngIf="selectedTasks.length === 1" translate>select.mode.count.singular</span>
+        <span *ngIf="selectedTasks.length !== 1" translate [translateParams]="{ number: selectedTasks.length || 0 }">select.mode.count.plural</span>
+      </div>
+      <ul class="unfiltered" *ngIf="(!loading || appending) && !filtered">
+        <mm-content-row
+          *ngFor="let task of tasksList; trackBy: listTrackBy"
+          [id]="task._id"
+          route="tasks"
+          [icon]="task.icon"
+          [heading]="task.heading"
+          [date]="task.due_date"
+          [summary]="task.summary"
+          [showStatus]="true"
+          [valid]="task.valid"
+          [verified]="true"
+          [statusIcon]="task.statusIcon"
+          [lineage]="task.lineage"
+          [unread]="!task.read"
+        >
+        </mm-content-row>
+      </ul>
+      <ul class="filtered" *ngIf="(!loading || appending) && filtered"></ul>
+      <p *ngIf="error" class="alert alert-danger" role="alert">
+    <span *ngIf="errorSyntax">
+      <span translate>invalid.query</span>
+    </span>
+        <span ng-hide="errorSyntax" translate>Error fetching tasks</span>
+      </p>
+      <p class="loading-status" *ngIf="!error && !loading && !hasTasks" translate>tasks.none</p>
```

Comment:
It looks like you're not using the same translations as the old template. https://github.com/medic/cht-core/blob/master/webapp/src/templates/directives/tasks_list.html#L7 
Can you please use those? As before, please migrate the angular 1.x templates instead of copying the Angular 10 report component template.